### ISSUE TITLE
[ui] New items notification not cleared when new items all confirmed

### ISSUE THREAD

**Issue by madduck** [eikek/docspell#2276]
The notification at the top of the screen alerts me to new documents:

![image](https://github.com/eikek/docspell/assets/195073/b0d6f8c4-b5e4-4298-8384-e65e669b0856)

Unless I click it, there is no way to get rid of it. Not even confirming the metadata of all new documents erases the notification. Say I was busy in another window while my colleague processed the new files, coming back to Docspell, I would see the notification, but I would not be able to find any new documents.

Either the notification needs to be cleared for all users when the new documents have been processed, or maybe there's a way to disable it altogether?

**Comment by madduck** [comment]
Similarly, the `inbox:yes` search tile will still list documents, even after they've been processed. This relates to #2277 I guess.

**Comment by eikek** [comment]
> the `inbox:yes` search tile will still list documents, even after they've been processed

Not sure if I understand: with "processed" do you mean someone clicked the "confirmed" button?

**Comment by madduck** [comment]
> > the `inbox:yes` search tile will still list documents, even after they've been processed
> 
> Not sure if I understand: with "processed" do you mean someone clicked the "confirmed" button?

Yes, someone clicked the confirm button. Unless I reload the page (why would I?), I still see the document as "new", when it isn't.

Contrast this to the fact that new documents *do appear* in the list as they appear in the system. So IMHO they should also disappear as their status changes.

**Comment by eikek** [comment]
I see, thanks! Yes - that is more or less the same as #2277

**Comment by madduck** [comment]
You know the UI better than I do, but yes. The header-pane notification area and each search tile should probably subscribe to all events and update themselves if required.
